sandbox/api/app.py

```diff
@@ -1,9 +1,36 @@
 from logging import INFO, basicConfig, getLogger
 from typing import Union
+from json import dumps
 
-from flask import Flask, request
+from flask import Flask, request, Response
 
-from .utils import check_for_errors, get_response
+from .utils import check_for_errors, load_json_file
+
+NOT_FOUND = "./api/responses/GET_RelatedPerson/not_found.json"
+EMPTY_RESPONSE = "./api/responses/GET_RelatedPerson/empty_response_9000000033.json"
+LIST_RELATIONSHIP = (
+    "./api/responses/GET_RelatedPerson/list_relationship_9000000017.json"
+)
+LIST_RELATIONSHIP_INCLUDE = (
+    "./api/responses/GET_RelatedPerson/list_relationship_include_9000000017.json"
+)
+VALIDATE_RELATIONSHIP_009 = (
```

Comment:
Would it be helpful to somehow make these identifiers more descriptive? I understand that they are referring to specific NHS numbers, but it would be helpful to make it easier to see immediately what is being checked.

Reply:
Can you give an example of what you mean?
Validate_relationship is the requested operation in this case.

Reply:
I mean using _009 etc rather than "_mother" or "_child" for example.

Reply:
The 009 and 025 are both child.  The current names reflect the overall operation, ie, List relationship or validate a relationship not the relationship itself.

I do not think this would add clarity.